2018-06-12T09:46:49.462Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'EMA' as the object already exists)
    at /home/raj_dhamsaniya/GitHub/AgriInput/agricultureinput-network/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-12T09:46:49.464Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'EMA' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'EMA' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/raj_dhamsaniya/GitHub/AgriInput/agricultureinput-network/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-12T09:46:49.826Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)
    at /home/raj_dhamsaniya/GitHub/AgriInput/agricultureinput-network/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-12T09:46:49.827Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/raj_dhamsaniya/GitHub/AgriInput/agricultureinput-network/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-12T09:47:06.481Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Instance 12345 has a property named disputeResolve which is not declared in org.acme.agriinputnetwork.Order)
    at /home/raj_dhamsaniya/GitHub/AgriInput/agricultureinput-network/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-12T09:47:06.481Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Instance 12345 has a property named disputeResolve which is not declared in org.acme.agriinputnetwork.Order)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Instance 12345 has a property named disputeResolve which is not declared in org.acme.agriinputnetwork.Order)\n\n    at _initializeChannel.then.then.then.then.catch (/home/raj_dhamsaniya/GitHub/AgriInput/agricultureinput-network/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
